.github/scripts/update_versions.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+import sys
+import pathlib
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+
+import semver
+
+from ruamel.yaml import YAML
+
+
+yaml = YAML(typ='rt')
+yaml.indent(mapping=2, sequence=4, offset=2)
+PREFIX = pathlib.PurePath('charts', 'netdata')
+
+
+def _parse_command_line():
+    """Defines arguments for the program and parses the given command line"""
+    parser = ArgumentParser(
+        formatter_class=RawDescriptionHelpFormatter,
+        description=('''\
+             Specify  arguments as listed below
+             Examples:
+             --------------------------------
+                python update_version.py get_chart_version
+                python update_version.py set_chart_version
+                python update_version.py get_app_version
+                python update_version.py set_app_version --version v1.31.0
+    '''))
+
+    parser.add_argument('action',
+                        choices=['get_chart_version', 'set_chart_version',
+                                 'get_app_version', 'set_app_version'])
+    parser.add_argument(
+                '--version',
+                required=False,
+                type=str,
+                nargs='?',
+                help='Version to set.')
+    return parser.parse_args()
+
+
+class Versioning:
+    '''Getting current Chart and App versions and bumping to new ones.'''
+    def __init__(self, chart_data, action, version=None):
+        self.action = action
+        self.version = version
+        self.chart_data = chart_data
+
+    def run(self):
+        getattr(self, self.action)()
+
+    def get_chart_version(self):
+        print(self.chart_data['version'])
+
+    def set_chart_version(self):
+        old_chart_data = self.chart_data['version']
+        new_chart_version = semver.VersionInfo.parse(old_chart_data) \
+                                                     .next_version(part='patch')
+
+        FileHandler.write_readme(old_chart_data, new_chart_version)
+        self.chart_data['version'] = str(new_chart_version)
+        FileHandler.write_chart(self.chart_data)
```

Comment:
This method doesn't update AppVersion in the README (which could be updated by agent-pr workflow)

Reply:
I mean the badge

https://github.com/netdata/helmchart/blob/6dc7e8fcdfc8c58815f3de7332de9335cd721ac0/charts/netdata/README.md?plain=1#L3

Reply:
True i will add it